third_party/windows/godot-cpp-windows.cmake

```diff
@@ -0,0 +1,219 @@
+# Windows-specific godot-cpp configuration (temporary fix)

```

Comment:
I'd like to not accept a temporary fix into the main repo as it'll then just be here forever...

Reply:
I should have probably dvelved into more an yeah that temporary fix is a stretch but i found out that the main culprit is the cmakelist from the godot-cpp folder, and i don't know if they may release a better cmake which is also suitable for windows uses in the future that is why i put this in the comment

Reply:
This is going to be really annoying to maintain as each specific godot-cpp update *might* break our build, and then I wouldn't notice it until someone tries using MSVC the next time and finds it broken.

If at all possible it would be better to try to modify any variables this script uses or modify the resulting targets it exports, so that any forceful customizations could still be kept outside this file (so that we can use the upstream file as-is so that we don't have to forever maintain our separate version).

Though one more alternative would be to open an issue / talk to the godot-cpp repo people and ask if they would accept changes to their file that make it MSVC compatible.